### PR TITLE
 docker: add capabilities and privileged support

### PR DESCRIPTION
**Pull request by artur-rs**
Added options for running container with:
* privileged mode
* all Linux capabilities added/dropped
* add/drop specific Linux capabilities (multiple)

Implements https://github.com/cockpit-project/cockpit/issues/10637

### CODE REVIEW

**Comment by KKoukiou** [comment]
@artur-rs this still needs a test in test/veify/check-docker in order to get merged. Are you fine to work on it?

**Comment by artur-rs** [comment]
@KKoukiou yes, I have tests ready, but the test architecture were a little problematic, so the tests weren't fully verified locally. I can push the code though.

> I am ok with the checkboxes, "Add ALL" "Drop ALL" but what's the default state? Some capabilities are added?

Yes, by default some of the capabilities are added. Full list can be found here: [url](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities)

**Comment by artur-rs** [comment]
@KKoukiou rebased and resolved merge conflicts

**Comment by marusak** [comment]
Needs to rebase to master since #12367 changed tests names

File: test/verify/check-docker [hunk 1]
```diff
@@ -404,6 +404,143 @@ CMD ["/bin/sh"]
...
+
+        # Run it and add/drop set of capabilities
+        b.click("#containers-images tr:contains(\"%s\") button.fa-play" % (image_name))
+        b.wait_popup("containers_run_image_dialog")
+        b.set_val("#containers-run-image-name", "PROBE7")
+        b.set_val("#containers-run-image-command", "/bin/sh")
+
+        # add ALL capabilities and drop specific one
+        b.click("#add-all-capabilities")
+        b.click("claim-capabilities")
```
Comment (marusak) [inline test/verify/check-docker]: Have you figured out how to run tests? This needs `#` as it is `id`. I havn't run anymore tests.

**Comment by marusak** [comment]
@artur-rs ping, are you block on something that I may help you with?

**Comment by marusak** [comment]
We [announced](https://cockpit-project.org/blog/cockpit-215.html) that we are deprecating cockpit-docker, so this is obsolete. Moving the underlying issue into cockpit-podman.

**Pull request closed without merging**